client/src/components/forms/field-error.component.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { describe, expect, it } from 'vitest';
+import { FieldError } from './field-error.component';
+import { render, screen } from '~/test/utils';
+
+describe('form-error', () => {
+  it('should render field-error on the page', () => {
+    render(<FieldError>Test Error</FieldError>);
+    expect(screen.getByText('Test Error')).toBeInTheDocument();
+  });
+  it('should render field-error with proper font size and message colour', () => {
+    render(<FieldError>Test Error</FieldError>);
+    expect(screen.getByText('Test Error')).toHaveClass('text-sm text-red-600');
+  });
```

Comment:
I don't think this 2nd test is that useful, testing using testing-library is meant to emulate what a user does. A user doesn't look for specific classnames, they look to interact with things so they look for elements like buttons, textboxes, or labels like Email and Password, to find what they need and to get info back on what they are to do.

If you really want to test styling, then you should consider `snapshots`, for instance, what if we replaced tailwind with material, so the app to the user, looks exactly the same. Then we would need to re-write many tests to change the styling tests. Snapshots take a snapshot and compare, if we did change, the test would fail and the system would ask us if we wanted to update our snapshot, which we would and then it would pass again.

Here are some blogs by Kent Dodds for you to read on what you should do in testing.

* https://kentcdodds.com/blog/common-mistakes-with-react-testing-library
* https://kentcdodds.com/blog/how-to-know-what-to-test
* https://kentcdodds.com/blog/testing-implementation-details

This test would fall under testing of implementation detail.